index.html

```diff
@@ -1,4 +1,172 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"
+      xmlns="http://www.w3.org/1999/html">
+<head>
+    <link rel=stylesheet href="styles.css">
+    <meta charset="UTF-8">
+    <title>Rekupe</title>
+</head>
+<body>
+<header>
+    <nav class="nav">
+        <ul>
+            <li><a href="#work">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#formulier">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="home">
+        <h1> Your website is an investment in the success of your business.</h1>
+        <h4>We are here to help you succeed.</h4>
+        <button type="button">Let's work together</button>
+    </div>
+    <img id=squiggleone src="assets/squiggle_line_stripe.svg" alt="squiggleone" width="3%">
+</header>
+
 <main>
+    <section class="intro">
+        <div class="one">
+            <h2>Rekupe is a digital agency in Los Angeles.</h2>
+            <h2>We use strategic UX, SEO optimization and beautiful visual designs to help
+                your business grow.</h2>
+            <h2> We see every project as a fresh opportunity, pushing ourselves to create new digital
+                experiences.</h2>
+            <br>
+            <p>Have a question? <a href="#formulier"> Click here</a> to contact us!</p>
+
+        </div>
+    </section>
+
+    <a id="work"></a>
+    <section class="middel">
+        <article class="articleone">
+            <div class="rollingstone"><h3>Rolling Stone</h3><img src="assets/squiggle_line.svg" alt="squiggleline"
+                                                                 width="20%">
+                You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.
+                <a href="emptypage.html" target="_blank">View project</a>
+            </div>
+            <img id="photodesk" src="assets/work-image-01.jpeg" alt="desk-with-plant"/>
+        </article>
+
+        <article class="articletwo">
+            <div class="variety">
+                <h3>Variety</h3><img src="assets/squiggle_line.svg" alt="squiggleline" width="20%">
+                Variety is an industry leader for breaking entertainment news, reviews and box office
+                results with
+                archive of content dating back to 1905.
+                <a href="emptypage.html" target="_blank">View project</a></div>
+            <figure><img src="assets/work-image-02.jpeg" alt="desk-with-book"></figure>
```

Comment:
Goed dat je overal een `alt` bij zet! _Tip_: `<figure>` is super netjes, maar zonder `<figcaption>` is hij overbodig